prime-router/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml

```diff
@@ -1,6 +1,6 @@
 extends: ../default-sender-transform
 constants:
-  serviceRequest:  'Bundle.entry.resource.ofType(ServiceRequest)'
+  serviceRequest: 'Bundle.entry.resource.ofType(ServiceRequest)'
   diagnosticReport: 'Bundle.entry.resource.ofType(DiagnosticReport)'
```

Comment:
@victor-chaparro Can you confirm that we never expect to have more than 1 ServiceRequest or DiagnosticReport or Patient from SR? I think these constants not having indices means we'd only apply the sender transforms to the first instance of each of these; if we had multiple, we'd need to split things out into sub-schemas.